README.md

```
@@ -57,6 +57,85 @@ Build requirements:
+
+To override the global file location:
+
+```properties
+mimir.resolvingLog.path=/path/to/my-resolving-log.csv
+```
+
+### Format
+
+The default format is CSV with a header row:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)